### DISCUSSION TITLE
Website design

### DISCUSSION THREAD

**Discussion by gazzabwoy** [gazzabwoy/projectEprom#4] [Ideas]

**Comment by kyleofawesome** [comment]
After going through the pages you made and showing us two different designs I say we just use this style of linking as its extremely simple and overall easy to read

![image](https://user-images.githubusercontent.com/35610708/138700426-0a43f2c2-832b-43f2-b5e2-a2697880e524.png)

**Reply by gazzabwoy** [reply]
> After going through the pages you made and showing us two different designs I say we just use this style of linking as its extremely simple and overall easy to read
> 
> ![image](https://user-images.githubusercontent.com/35610708/138700426-0a43f2c2-832b-43f2-b5e2-a2697880e524.png)

Do you agree with kyle  @fagann ? @marie ?

**Comment by billion07** [comment]
github I thing is more easier

**Reply by billion07** [reply]
yes

**Comment by gazzabwoy** [comment]
![image](https://user-images.githubusercontent.com/91916706/139559505-f3799806-5b61-4ed0-99a2-f89513abd660.png)

I'm using this type of menu for my project, its same as this project menu, just different colours
